www/src/examples/AsPropAccordion.js

```diff
@@ -0,0 +1,14 @@
+<div>
+  <h1>
+    Example heading{' '}
+    <Badge bg="secondary" as="Button">
+      New
+    </Badge>
+  </h1>
+  <h2>
+    Example heading{' '}
+    <Button bg="secondary" as="Badge">
```

Comment:
Currently Buttons can only accept regular tags at the moment because we need to apply the correct button props in `@restart/ui`.

Let's leave out the button example and keep the badge example.

Reply:
@kyletsang have removed Button example as suggested and even changed the file name to more appropriate name.